### ISSUE TITLE
Errors after subscribing to soco.services.Queue

### ISSUE THREAD

**Issue by relevitt** [SoCo/SoCo#378]
I'm using soco. 0.11.1, installed via pip.  This code ...

```
import soco
from pprint import pprint
from Queue import Empty
device = list(soco.discover())[0]
group = device.group
device = group.coordinator
service = soco.services.Queue(device)
sub = service.subscribe()
while True:
    try:
        event = sub.events.get(timeout=1)
        if event.variables:
            pprint(event.variables)
    except Empty:
        pass
```

generates this error ...

```
~$ python soco_test.py 
----------------------------------------
Exception happened during processing of request from ('192.168.1.86', 53098)
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 593, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 649, in __init__
    self.handle()
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 328, in handle_one_request
    method()
  File "/usr/local/lib/python2.7/dist-packages/soco/events.py", line 230, in do_NOTIFY
    variables = parse_event_xml(content)
  File "/usr/local/lib/python2.7/dist-packages/soco/events.py", line 116, in parse_event_xml
    for last_change_var in instance:
TypeError: 'NoneType' object is not iterable
----------------------------------------
```

Don't think it's relevant, but I've commented out a section of data_structures.py (lines 356-362) as follows ...

```
        # Check we have the right sort of element. tag can be an empty string
        # which indicates that any tag is allowed (see eg the musicAlbum DIDL
        # class)
#        if not element.tag.endswith(cls.tag):
#            raise DIDLMetadataError(
#                "Wrong element. Expected '<{0}>',"
#                " got '<{1}>'".format(cls.tag, element.tag))

```

FWIW, I'm trying to detect changes to a sonos queue. I can detect these by subscribing to each visible device's avTransport service and testing for events where the change in event.variables can't otherwise be explained. Whenever a track is moved in the queue, an event will be sent on the avTransport service of one of the visible devices. However, the event.variables give no indication what the event was. That's my current way of detecting changes in the queue! I was hoping there might be a more direct way of doing this with events.

**Comment by nlagaros** [comment]
I'm not sure if you have figured this out yet, but I do something similar and made a small addition in events.py to handle LastChange from the queue.  The namespace is different for the queue so I added this:
```
if instance is None:
    instance = last_change_tree.find("{urn:schemas-sonos-com:metadata-1-0/Queue/}QueueID")
```
right after the instance check for RCS - around line 109 in events.py.

**Comment by cassieramen** [comment]
Thanks @nlagaros 
That fixed it for me.

**Comment by relevitt** [comment]
Thanks.

**Comment by bosim** [comment]
Anybody have an idea when we will see a fix *upstream*?

**Comment by ghcs27** [comment]
This was (hopefully) resolved by #467. If it still doesn't work for you, feel free to reopen.
